diktat-rules/src/test/resources/test/paragraph3/else_expected/ElseInWhenExpected.kt

```diff
@@ -0,0 +1,16 @@
+package test.paragraph3.else_expected
+
+enum class TestEnum {
+    ONE, TWO
+}
+
+fun testWhenExpression() {
+    val directoryType = TestEnum.ONE
+
+    val objectType = when (directoryType) {
+        TestEnum.ONE -> "d"
+        TestEnum.TWO -> "-"
+else -> {
```

Comment:
This fix doesn't make sense to me, see another comment. I'm surprised that compiler allows `else` branch when choice is exhaustive. Please add another example with `when` statement.

Reply:
I am even afraid that compiler is so stupid that it will raise a warning even in this case